packages/react-table/src/demos/Table.md

```diff
@@ -3076,20 +3076,28 @@ class LoadingStateDemo extends React.Component {
 
 ### Error
 
-```js
+```js isFullscreen
 import React from 'react';
-import { EmptyState, EmptyStateBody, EmptyStateVariant, Title } from '@patternfly/react-core';
+import {
+  Bullseye,
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  PageSection,
+  Title
+} from '@patternfly/react-core';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
```

Comment:
This is already imported you can remove this here and in any other Table PRs that are open that might have this.

Reply:
If I try to remove this I get a lint error that prevents me from commiting/pushing! I think the build issue was due to this being declared twice at the top of the file -- it's only declared once now 👍